chart/k8gb/templates/coredns-cm.yaml

```diff
@@ -18,6 +18,7 @@ data:
             resources DNSEndpoint
             filter k8gb.absa.oss/dnstype=local
             negttl {{ .Values.k8gb.dnsZoneNegTTL }}
+            loadbalance weight
```

Comment:
isn't it hardcoding particular method?

Reply:
Would be great to get confirmation from @kuritka , but if I understand correctly it does standard round-robin if no WRR in Gslb spec, see https://github.com/k8gb-io/coredns-crd-plugin/pull/45

Reply:
Hi @ytsarev, RR is only a subset of WRR.  If weights are not defined , the RR is automatically applied.  If weights are defined in CR, the WRR is used. If an error occurs for any reason, then the RR is used automatically. So there should stay `loadbalance weight `. 

The reason why I defines this field is for extendability. I was planning that in the future there will be a `loadbalance consistent` etc.

Reply:
Perfect, thank you!